### DISCUSSION TITLE
Wifi access point mode

### DISCUSSION THREAD

**Discussion by Doncode** [chris1111/Wireless-USB-Big-Sur-Adapter#239] [General]
Hi. Is it possible to use external adapter as Wifi access point mode?

**Comment by chris1111** [comment]
This is a discussion

**Comment by chris1111** [comment]
[View this](https://mackeeper.com/blog/turn-your-mac-into-a-wi-fi-hotspot/)

**Comment by a9eesh** [comment]
Page not found

**Comment by chris1111** [comment]
![Screenshot 2023-07-27 at 19 13 26](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/assets/6248794/14b8d2c3-e57e-4215-b627-f6a9233d0760)
![Screenshot 2023-07-27 at 19 14 54](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/assets/6248794/26001881-7e1f-4430-858c-264e3ab56fbe)
![Screenshot 2023-07-27 at 19 15 53](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/assets/6248794/7ddaacb9-725e-4fd8-b61b-8a51e4c1539b)
![Screenshot 2023-07-27 at 19 16 28](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/assets/6248794/7bac0217-5877-4947-907e-a946d94d25e9)
![Screenshot 2023-07-27 at 19 16 53](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/assets/6248794/7028a1f3-39d1-43eb-8b32-1cb746d8d9b7)

**Reply by yemix** [reply]
When i try to to use the external for creating a access point it down show up cause its not a „real“ wifi adapter, when i use my integrated it works fine, somestimes no internet access.
